src/routers/features/siciliaVola/secured/index.ts

```diff
@@ -122,7 +122,7 @@ addHandler(
         v => v.idVoucher !== maybeVoucherId.value
       );
     }
-    res.sendStatus(200);
+    res.status(200).json({});
```

Comment:
```suggestion
    res.json({});
```
`json` is a shortcut to response with status 200 and a json payload